src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -104,6 +110,8 @@ def load_arguments(self, _):
         c.argument('partition_key_path', options_list=['--partition-key-path', '-p'], help='Partition Key Path, e.g., \'/address/zipcode\'')
         c.argument('default_ttl', options_list=['--ttl'], type=int, help='Default TTL. If the value is missing or set to "-1", items don’t expire. If the value is set to "n", items will expire "n" seconds after last modified time.')
         c.argument('indexing_policy', options_list=['--idx'], type=shell_safe_json_parse, completer=FilesCompleter(), help='Indexing Policy, you can enter it as a string or as a file, e.g., --idx @policy-file.json or ' + SQL_GREMLIN_INDEXING_POLICY_EXAMPLE)
+        c.argument('unique_key_policy', options_list=['--unique'], type=shell_safe_json_parse, completer=FilesCompleter(), help='Unique Key Policy, you can enter it as a string or as a file, e.g., --unique @policy-file.json or ' + SQL_UNIQUE_KEY_POLICY_EXAMPLE)
```

Comment:
There are two problems here: 1. --unique is a little strange, can you change to another one? 2. Can you also provide full argument name in your options_list like `partition_key_path`?

The same for conflict_resolution_policy.